Source/Spring2022_Capstone/PlayerCharacter.cpp

```diff
@@ -205,6 +205,21 @@ void APlayerCharacter::TakeHit()
 	}
 }
 
+void APlayerCharacter::IncreaseMaxHealth(int MaxHealthIncrease)
+{
+	HealthComponent->SetMaxHealth(HealthComponent->GetMaxHealth() + MaxHealthIncrease);
+	HealthComponent->SetHealth(HealthComponent->GetHealth() + MaxHealthIncrease);
+	GEngine->AddOnScreenDebugMessage(0, 4.f, FColor::Red, FString::Printf(TEXT("Your new max health is: %f"), HealthComponent->GetMaxHealth()));
+}
+
+void APlayerCharacter::IncreaseMaxHealthPercentage(int Percentage)
```

Comment:
Could maybe change Percentage to PercentageIncrease to match other function.

Reply:
I did percentage / value instead so it's more consistent that way but it's on the next PR